zip-0315.rst

```diff
@@ -4,6 +4,315 @@
   Title: Best Practices for Wallet Handling of Multiple Pools
   Owners: Daira Hopwood <daira@electriccoin.co>
           Jack Grigg <jack@electriccoin.co>
-  Status: Reserved
+  Status: Proposed
   Category: Wallet
   Discussions-To: <https://github.com/zcash/zips/issues/447>
+  Pull-Request: <https://github.com/zcash/zips/issues/>
+
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", and "MAY" in this document are to be
+interpreted as described in RFC 2119. [#RFC2119]_
+
+The terms below are to be interpreted as follows:
+
+Auto-shielding
+    The process of automatically transferring transparent funds to the most recent
+    shielded pool on receipt.
+
+Auto-migration
+    The process of automatically transferring shielded funds from older pools to the
+    most preferred (usually the most recent) pool on receipt.
+
+TODO: add definitions of opportunistic shielding and migration.
+
+TODO: "confirmed", "partially confirmed"
+
+
+Motivation
+==========
+
+The aim of this ZIP is to provide wallet developers with a set of best practices by
+which they can minimize the leakage of information when constructing transactions.
```

Comment:
```suggestion
which they can minimize the leakage of information and maximize their privacy when
constructing transactions.
```

Reply:
The goals being to have security and privacy best practices supported by default when handling funds and transactions as the user's agent, so that the user is not obliged by default to juggle all of the things.